lib/makeup/lexers/erlang_lexer.ex

```diff
@@ -167,24 +167,12 @@ defmodule Makeup.Lexers.ErlangLexer do
   syntax_operators =
     word_from_list(~W[+ - +? ++ = == -- * / < > /= =:= =/= =< >= ==? <- ! ? ?!], :operator)
 
-  define =
-    token("define", :name_entity)
+  module_attribute =
+    token("-", :punctuation)
     |> optional(whitespace)
-    |> concat(token("(", :punctuation))
-    |> concat(token(macro_name, :name_constant))
-
-  record =
-    token("record", :name_entity)
+    |> concat(atom_name |> token(:name_attribute))
     |> optional(whitespace)
     |> concat(token("(", :punctuation))
-    |> concat(token(macro_name, :name_label))
-
-  normal_directive = token(atom_name, :name_entity)
-
-  # We need to add a newline to the file for this to work
-  directive =
-    token("\n-", :punctuation)
```

Comment:
I removed the match on this newline as it was not working with it and I was not able to find an explanation to why this is here

Reply:
This is to make sure that we only match the beginning of the line. For this to work, the lexer must add a newline to the file before starting to lex it.

Reply:
But why we need to make sure only to match the beginning of the line? isn't that way too _restrictive_? makes sense to be too restrictive for a highlighter? 🤔

Reply:
FWIW, Erlang does allow spaces between the newline and the directive.

Reply:
So what would be the best course of action here? 🤔 
The [way it is now](https://github.com/tmbb/makeup_erlang/pull/9/files#diff-fc83a22ef522afa817de5718af0054caR171), it matches with or without the newline

Reply:
> But why we need to make sure only to match the beginning of the line?

Yes.

Otherwise, the following code would be lexed as if it were an attribute:

```elixir
a(X) - b(Y)
```

Reply:
So should we do it newline, any spaces and then minus?

Reply:
> So should we do it newline, any spaces and then minus?

I think so, but you're the Erlang expert xD I haven't written much Erlang

Reply:
Hmmm, I see. Makes _a lot_ of sense. I will add the match on the newline again also being permissive about spaces between the newline and the `-`, wdyt?

Reply:
> Hmmm, I see. Makes a lot of sense. I will add the match on the newline again also being permissive about spaces between the newline and the -, wdyt?

I agree with this

Reply:
[Fixed here](https://github.com/tmbb/makeup_erlang/compare/d983cdc8da607d0a16c5694dff0d79f6199fb993..2c05b4ab7027413351936c5e39556ed6000c19d4). What do you folks think about this implementation?